### DISCUSSION TITLE
Pos tag bug when processing contiguous emojis

### DISCUSSION THREAD

**Discussion by charx7** [explosion/spaCy#10276] [Help: Other Questions] [answered]
## Problem
After loading the `es_core_news_sm` library for for an nlp pipeline, and trying read the POS of a document that contains contiguous emojis the detected pos_ tag changes depending on the number of contiguous emojis

## How to reproduce the behaviour
```python
import spacy
import es_core_news_sm

# init spacy nlp
nlp = es_core_news_sm.load()

s1 = "que tenga un excelente fin de año líder y que este y los que sigan sean mejores 👏🙏"
s2 = "que tenga un excelente fin de año líder y que este y los que sigan sean mejores 👏🙏🙏"

doc = nlp(s1)
for word in doc:
  print(f"The word: {word} has a pos tag {word.pos_}")
# >> observe that the POS tag of the 🙏 is PUNCT

doc = nlp(s2)
for word in doc:
  print(f"The word: {word} has a pos tag {word.pos_}")
# >> observe that the POS tag of the 🙏 is PROPN
```
We can observe that depending the number of contiguous 🙏 emojis the POS tag changes :( 

## Your Environment

* *spaCy version:* 2.2.4
* *Platform:* Linux-5.4.144+-x86_64-with-Ubuntu-18.04-bionic
* *Python version:* 3.7.12
* *Models:* en
* Environment Information: google-collab

**Comment by polm** [comment, marked as answer]
Please see #3052 - the statistical models make mistakes sometimes, and we can't address individual errors. 

Also note that spaCy v2.2.4 is pretty old at this point - v3 has been out over a year, and I'd recommend you give it a look.

**Reply by charx7** [reply]
Thanks, I will bump the version of spaCy and try again
